eng/targets/DiaSymReaderNative.targets

```diff
@@ -18,7 +18,7 @@
     package can't be referenced directly but rather has to have it's assets manually copied 
     out. This logic is responsible for doing that.
   -->
-  <ItemGroup Condition="'$(DotNetBuildFromSource)' != 'true'">
+  <ItemGroup>
```

Comment:
This isn't legal in source build so surprised to see this change. Didn't see another obvious change which excludes this so can you elaborate on how this should work?

Reply:
Why wouldn't this be legal for source builds? It's my understanding that contents of packages referenced in a source build environment are source build binaries, so the existence/availability of a binary at this location in the build means the binary is available in source build.

Reply:
I removed these two binaries from the source-built toolset packages.